support/describe-operation-hooks.js

```diff
@@ -12,12 +12,12 @@ var Memory = require('../lib/connectors/memory').Memory;
 var HOOK_NAMES = [
   'access',
   'before save', 'persist', 'loaded', 'after save',
-  'before delete', 'after delete'
+  'before delete', 'after delete',
```

Comment:
In this place, I am grouping logically-related hooks together. Also note that the order here dictates the order of columns in the output table, where I definitely want to see hooks listed in an order that makes it easy to compare context properties reported by related hooks.